Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -5,9 +5,17 @@
 import re
 import xlrd
 import csv
+import tldextract
+import warnings
+import traceback
 
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
```

Comment:
Why is this here?

Reply:
The use of tldextract package raises SSL errors. this is used to disable these errors (same as in JSONFeedApiModule where I took the Domain recognition from)